### DISCUSSION TITLE
Netflix check if protected content is on

### DISCUSSION THREAD

**Discussion by rayydev** [qutebrowser/qutebrowser#7377] [Q&A] [answered]
When i try to watch netflix on qutebrowser i have a error that says to check if in chrome://settings/content/protectedContent is on and i dont know how to fix it

**Comment by The-Compiler** [comment, marked as answer]
Please use discussions for questions, not issues - I converted this to one.

Can you please show your `:version` information? Does it happen with `--temp-basedir`?

**Reply by rayydev** [reply]
:version = https://paste.the-compiler.org/view/22b12d8b 

and it did not happen with --temp-basedir

**Reply by The-Compiler** [reply]
Probably something in your settings then. Can you show your `:config-diff`?

**Reply by rayydev** [reply]
it says Default configuration

**Reply by The-Compiler** [reply]
No idea then. Do you have `qt5-webengine-widevine` and/or `chromium-widevine` installed?

**Reply by rayydev** [reply]
I dont know. Do you mind to say me how i can check it?

**Reply by The-Compiler** [reply]
Just like any other package - on Arch, probably via `pacman -Qi`.

**Reply by rayydev** [reply]
It dosnt says thing about that. `Depends On      : python-jinja  python-pyqt5  python-yaml  qt5-base  python-pyqtwebengine
Optional Deps   : python-adblock: adblocking backend
                  python-pygments
                  gst-libav: media playback with qt5-webkit backend
                  gst-plugins-base: media playback with qt5-webkit backend
                  gst-plugins-good: media playback with qt5-webkit backend
                  gst-plugins-bad: media playback with qt5-webkit backend
                  gst-plugins-ugly: media playback with qt5-webkit backend
                  pdfjs: displaying PDF in-browser
                  qt5-webkit: alternative backend`

**Reply by rayydev** [reply]
also in the `:version` it says that is based on chromium so i really dont know

**Reply by rayydev** [reply]
Just fixed it. I Am downloaded the widevine and it worked! But thanks for helping you can close the disscusion

**Comment by abreathingcorpse** [comment]
Hello @The-Compiler, got the same issue but with Prime. I do not think that I've updated the configuration excpept by enabling the adblock. It should be the latest version, since pacman doesn't show me that qutebrowser can be updated.

How should I start debugging this?

**Reply by The-Compiler** [reply]
I know almost nothing about Widevine stuff. I suggest you open a new discussion and see if someone can suggest something. You might also want to try with Qt 6 (see #7202) and qt6-webengine-widevine installed perhaps.

**Reply by abreathingcorpse** [reply]
Sorry if this is a dumb question but, what is Widevine and does this have anything to do with Qutebrowser? I looked at the Arch Package List and couldn't find anything related to Widevine.

**Reply by mschilli87** [reply]
@abreathingcorpse: widevine is a DRM plugin that is required to play streamed content from Netflix, Amazon, and the likes. So it has nothing to do with qutebrowser (which is why this is the wrong place for you to learn more about it) other than qutebrowser - via the underlying webengine - being the software that you happen to use to open the stream. Decryption is not handled by qutebrowser but the widevine plugin for chromium which you need to install and configure for your qt-webengine installation that is also use by qutebrower. How you do this depends on your operating system and again has nothing to do with qutebrowser. If you can play a stream in Chromium, but not qutebrowser, than the issue could be on qutebrowser's end or on the Qt-layer (testing with a KDE-browser should tell you).

**Reply by abreathingcorpse** [reply]
Ah, I understand better know. Thank you for taking the time to clarify this. I'll have to read more regarding qt-webengine and widevine now.

**Comment by voxd2600** [comment]
wow

**Comment by tkudlicka** [comment]
Hey, i got same issue but on macos. i bee searching around that but couldnt find anything. Did somebody get work around that?
